web/modules/custom/dpl_login/dpl_login.module

```diff
@@ -295,29 +308,8 @@ function dpl_login_preprocess_html(array &$variables): void {
 }
 
 /**
- * Implements hook_form_FORM_ID_alter().
- */
-function dpl_login_form_openid_connect_admin_settings_alter(array &$form, FormStateInterface $form_state): void {
-  $form['#submit'][] = 'dpl_login_openid_connect_admin_settings_submit';
-}
-
-/**
- * Custom submit handler for openid_connect_admin_settings form.
- *
- * @param array<mixed> $form
- *   An associative array containing the structure of the form.
- * @param \Drupal\Core\Form\FormStateInterface $form_state
- *   The current state of the form.
- *
- * @throws \Drupal\dpl_login\Exception\MissingConfigurationException
+ * Helper function for checking if the plugin id is "adgangsplatformen".
  */
-function dpl_login_openid_connect_admin_settings_submit(array &$form, FormStateInterface $form_state): void {
-  $messenger = \Drupal::messenger();
-
-  /** @var Drupal\dpl_library_token\LibraryTokenHandler $handler */
-  $handler = Drupal::service('dpl_library_token.handler');
-
-  if (!$handler->retrieveAndStoreToken(TRUE)) {
-    $messenger->addMessage(t('Unable to retrieve token from Adgangsplatformen. Please check if the provided Agency ID is correct.', [], ['context' => 'OpenId Connect Settings']), LogLevel::ERROR);
-  }
+function dpl_login_check_if_plugin_id_is_adgangsplatformen(string $plugin_id): bool {
```

Comment:
Please reconsider naming and/or responsibility of this function.

1. The current function name is longer than the actual check. With this in mind it could be renamed `dpl_login_plugin_id_is_adgangsplatformen()`.
2. The function is about as safe as `$context['plugin_id'] === PLUGIN_ID_ADGANGSPLATFORMEN` with the string defined as a constant.
3. Another approach would be to increase the responsiblity of the function and accept a OpenID context array. The function would then check if the `plugin_id` entry has the expected value.